contracts/upgradeable_contracts/U_ForeignBridge.sol

```diff
@@ -215,11 +234,20 @@ contract ForeignBridge is ERC677Receiver, BasicBridge {
         emit DailyLimit(_foreignDailyLimit);
     }
 
+    function setHomeDailyLimit(uint256 _homeDailyLimit) external onlyOwner {
+        uintStorage[keccak256("homeDailyLimit")] = _homeDailyLimit;
```

Comment:
Does it make sense to emit an event similarly it is done in `setForeignDailyLimit`?

Reply:
Yes, added an event.